ci-operator/config/openshift/installer/openshift-installer-release-4.12.yaml

```diff
@@ -525,6 +525,16 @@ tests:
         AGENT_E2E_TEST_SCENARIO=COMPACT_IPV4
         AGENT_DEPLOY_MCE=true
     workflow: agent-e2e-generic
+- as: e2e-agent-sno
```

Comment:
from our latest team discussion, the new job name must be changed to `e2e-agent-compact-ipv4`